qiskit/optimization/converters/optimization_problem_to_qubo.py

```diff
@@ -45,14 +45,13 @@ def __init__(self, penalty: Optional[float] = 1e5) -> None:
         self._penalty = penalty
 
     def encode(self, problem: OptimizationProblem) -> OptimizationProblem:
-        """ Convert a problem with inequality constraints into new one with only equality
-        constraints.
+        """ Convert a problem with linear equality constraints into new one with a QUBO form.
 
         Args:
-            problem: The problem to be solved, that may contain inequality constraints.
+            problem: The problem with linear equality constraints to be solved.
 
         Returns:
-            The converted problem, that contain only equality constraints.
+            The converted problem whos form is QUBO  
```

Comment:
```suggestion
            The problem converted in QUBO format.
```

Reply:
Thank you for the comment. It's more than welcome to have comments for natural English since apparently English is not my first languages. I will change the comment as the above.